src/LBErrors.sol

```diff
@@ -97,7 +96,7 @@ error LBPair__NewSizeTooSmall(uint256 newSize, uint256 oracleSize);
 
 error BinHelper__BinStepOverflows(uint256 bp);
 error BinHelper__IdOverflows(int256 id);
-error BinHelper__IntOverflows(uint256 id);
+error BinHelper__IdOverflows(uint256 id);
```

Comment:
The compiler doesn't seem to like this, it's throwing me an error saying that the two `BinHelper__IdOverflows` errors are identical